src/notevennexus/io.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+import hashlib
+
+
+def compute_checksum(file_path) -> str:
+    checksums = {
+        'md5': hashlib.md5(usedforsecurity=False),
+        'sha256': hashlib.sha256(usedforsecurity=False),
```

Comment:
Because the day before I reported an issue in an file, and it was fixed, and I got a new file with the same name. I thought the checksums will allow for ensuring people talk about the same file, etc.